lib/perl/Genome/VariantReporting/Process/Trio.pm

```diff
@@ -140,7 +140,8 @@ sub get_cle_input_results {
 sub special_compare_functions {
     my $self = shift;
     my @functions = $self->SUPER::special_compare_functions(@_);
-    push @functions, qr(\.xml$) => sub {my ($a, $b) = @_; Genome::VariantReporting::Process::Trio::compare_igv_xml($a, $b)};
+    push @functions, qr(\.xml$) => sub {Genome::VariantReporting::Process::Trio::compare_igv_xml(@_)};
+    push @functions, qr(\.yaml$) => sub {return 0};
```

Comment:
If you wanted, you could have the first `push @functions` take both functions instead of repeating it for each one.